lib/index.js

```diff
@@ -81,6 +82,24 @@ const authenticateUser = (username, password) => {
     .catch(error => console.error({error}));
 }
 
+// ***** Register User *****
+
+const registerUser = (username, password, passwordConfirmation) => {
+  let formData = new FormData();
+
+  formData.append('email', username);
```

Comment:
Kinda confusing mixing email and username here

Reply:
I will edit this for better clarity thanks